client/src/main/java/io/pravega/client/segment/impl/SegmentAttribute.java

```diff
@@ -30,4 +30,6 @@
         HashHelper hash = HashHelper.seededWith("SegmentAttribute");
         value = hash.toUUID(this.name());
     }
+    
```

Comment:
These two new lines are unnecessary and without them, this file would not be changed at all. Please revert these.